homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -26,20 +26,24 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 
     all_devices = api.get_all_devices()
     for dev_id, device_properties in all_devices.items():
-        if device_properties["class"] == "heater_central":
-            data = api.get_device_data(dev_id)
-            for binary_sensor, dummy in BINARY_SENSOR_MAP.items():
-                if binary_sensor in data:
-                    entities.append(
-                        PwBinarySensor(
-                            api,
-                            coordinator,
-                            device_properties["name"],
-                            binary_sensor,
-                            dev_id,
-                            device_properties["class"],
-                        )
-                    )
+        if "heater_central" not in device_properties["class"]:
+            continue
+
+        data = api.get_device_data(dev_id)
+        for binary_sensor, dummy in BINARY_SENSOR_MAP.items():
+            if not binary_sensor not in data:
```

Comment:
One not too many? Ìt should read: `if binary_sensor not in data:

Reply:
True :)